melanges-de-linstitut-dominicain-detudes-orientales-en.csl

```diff
@@ -0,0 +1,260 @@
+    <link href="http://www.zotero.org/styles/melanges-de-linstitut-dominicain-detudes-orientales-en" rel="self"/>
+    <link href="https://www.ideo-cairo.org/fr/mideo-fr/le-mideo-les-melanges-de-lideo/" rel="documentation"/>
+    <author>
+      <email>souchon.nicolas.ns@gmail.com</email>
+      <name>Nicolas Souchon</name>
+    </author>
+    <category citation-format="note"/>
+    <category field="history"/>
+    <summary>Style sheet for the collection Mélanges de l'Institut dominicain d'études orientales (MIDÉO)</summary>
+    <updated>2024-09-25T12:19:45+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="fr">
+    <terms>
+      <term name="no-place" form="short">n. p.</term>
+      <term name="no-publisher" form="short">n. n.</term>
+      <term name="no date" form="short">n. d.</term>
+      <term name="in">in</term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author">
+      <name delimiter-precedes-et-al="never" delimiter-precedes-last="never" initialize="false" name-as-sort-order="all"/>
+    </names>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="book" match="any">
+        <text variable="title" font-style="italic"/>
+      </if>
+      <else>
+        <text variable="title" quotes="true" font-style="normal"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="locator">
+    <text variable="locator"/>
+  </macro>
+  <macro name="container-title">
+    <text variable="container-title" font-style="italic"/>
+  </macro>
+  <macro name="volume">
+    <number variable="volume"/>
+  </macro>
+  <macro name="issue">
+    <number variable="issue"/>
+  </macro>
+  <macro name="publisher-place">
+    <choose>
+      <if match="any" variable="publisher-place">
+        <text variable="publisher-place"/>
+      </if>
+      <else>
+        <text term="no-place" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <choose>
+      <if match="any" variable="publisher">
+        <text variable="publisher"/>
+      </if>
+      <else>
+        <text term="no-publisher" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="volume-number">
+    <group delimiter=" ">
+      <number variable="number-of-volumes"/>
+      <label variable="volume" form="short"/>
+    </group>
+  </macro>
+  <macro name="issued">
+    <choose>
+      <if match="any" variable="issued">
+        <date date-parts="year" form="numeric" variable="issued"/>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="original-date">
+    <date date-parts="year" form="numeric" variable="original-date" prefix="[" suffix="]"/>
+  </macro>
+  <macro name="url">
```

Comment:
```suggestion
  <macro name="date">
    <choose>
      <if match="any" variable="original-date">
        <group delimiter="/">
          <text macro="original-date"/>
          <text macro="issued" prefix="[" suffix="]"/>
        </group>
      </if>
      <else>
        <text macro="issued"/>
      </else>
    </choose>
  </macro>
  <macro name="url">
```

Reply:
A few corrections need to be made here. There is no / between the original-date and the issued, which is used in the recommendations to separate the years of the hegira from those of the Christian calendar. Next, the original-date should appear in square brackets after the issued